newIDE/app/src/Utils/GDevelopServices/Tutorial.js

```diff
@@ -56,9 +64,26 @@ export const getInstructionTutorialIds = (type: string): Array<string> => {
     case 'EcrireFichierTxt':
     case 'LireFichierExp':
     case 'LireFichierTxt':
-      return ['save-and-load'];
+      return ['intermediate-storage\n', 'advanced-saving-loading-storage'];
```

Comment:
```suggestion
      return ['intermediate-storage', 'advanced-saving-loading-storage'];
```

Reply:
Actually, the `\n` is what is returned by our API. Should it be changed directly "in base" ?

Reply:
I can format the fields to avoid newlines, on the spreadsheet.